packages/token-lists/src/tokens/pancakeswap-top-100.json

```diff
@@ -5,798 +5,798 @@
+    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x8b303d5BbfBbf46F1a4d9741E491e06986894e18/logo.png"
   },
   {
     "name": "Paralink Network",
     "symbol": "PARA",
     "address": "0x076DDcE096C93dcF5D51FE346062bF0Ba9523493",
     "chainId": 56,
     "decimals": 18,
-    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x076ddce096c93dcf5d51fe346062bf0ba9523493/logo.png"
+    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x076DDcE096C93dcF5D51FE346062bF0Ba9523493/logo.png"
   },
   {
     "name": "DeChart",
     "symbol": "DCH",
     "address": "0x14ACe3d3429408bFA8562099A253172913AD71bd",
     "chainId": 56,
     "decimals": 18,
-    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x14ace3d3429408bfa8562099a253172913ad71bd/logo.png"
+    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x14ACe3d3429408bFA8562099A253172913AD71bd/logo.png"
   },
   {
     "name": "SafeMars 2",
     "symbol": "SAFEMARS",
     "address": "0x3aD9594151886Ce8538C1ff615EFa2385a8C3A88",
     "chainId": 56,
     "decimals": 9,
-    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x3ad9594151886ce8538c1ff615efa2385a8c3a88/logo.png"
+    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x3aD9594151886Ce8538C1ff615EFa2385a8C3A88/logo.png"
   },
   {
     "name": "SafePluto",
     "symbol": "SAFEPLUTO",
     "address": "0xF40153fb3F5DB22F6843fc63835A3A870Ed846d3",
     "chainId": 56,
     "decimals": 9,
-    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0xf40153fb3f5db22f6843fc63835a3a870ed846d3/logo.png"
+    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0xF40153fb3F5DB22F6843fc63835A3A870Ed846d3/logo.png"
   },
   {
     "name": "JaguarSwap Token",
     "symbol": "JAGUAR",
     "address": "0x4a3524936Db5C310d852266033589D3f6F30BA5d",
     "chainId": 56,
     "decimals": 18,
-    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x4a3524936db5c310d852266033589d3f6f30ba5d/logo.png"
+    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x4a3524936Db5C310d852266033589D3f6F30BA5d/logo.png"
   },
   {
     "name": "Mobox",
     "symbol": "MBOX",
     "address": "0x3203c9E46cA618C8C1cE5dC67e7e9D75f5da2377",
     "chainId": 56,
     "decimals": 18,
-    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x3203c9e46ca618c8c1ce5dc67e7e9d75f5da2377/logo.png"
+    "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x3203c9E46cA618C8C1cE5dC67e7e9D75f5da2377/logo.png"
   },
   {
     "name": "Wault",
     "symbol": "WAULTx",
-    "address": "0xB64E638E60D154B43f660a6BF8fD8a3b249a6a21",
+    "address": "0xB64E638E60D154B43f660a6BF8fD8a3b249a6a2",
```

Comment:
Hm.... 1 in the end got lost somehow, probably I haven't selected it when I copied the list to you